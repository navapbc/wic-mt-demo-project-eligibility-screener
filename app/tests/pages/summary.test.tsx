import { render, screen, waitFor } from '@testing-library/react'
import { axe } from 'jest-axe'

import Summary from '@pages/summary'

describe('Summary', () => {
  it('should render the heading', () => {
    render(<Summary />)

    const heading = screen.getByText(/Your confirmation/i)

    expect(heading).toBeInTheDocument()
    expect(heading).toMatchSnapshot()
  })

  it('should pass accessibility scan', async () => {
    const { container } = render(<Summary />)
    const results = await axe(container)

    await waitFor(() => {
      expect(results).toHaveNoViolations()
    })
  })
})
