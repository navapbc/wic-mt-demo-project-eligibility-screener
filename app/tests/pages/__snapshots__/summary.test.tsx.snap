// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Summary should render the heading 1`] = `
<h1>
  Your confirmation
</h1>
`;
